app/main.py

```diff
@@ -2,0 +1,19 @@
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3 and command.split()[0] == "mv":
+        mv, file1_name, destination = command.split()
```

Comment:
You are calling `command.split()` multiple times. According to the checklist, you should call it once and store the result in a variable.